webpack/assets/javascripts/react_app/components/Layout/components/UserDropdowns.js

```diff
@@ -21,32 +22,31 @@ const UserDropdowns = ({
     >
       <NotificationContainer data={{ url: notificationUrl }} />
     </NavItem>
-    {!!user &&
-      !!user.current_user && (
-        <NavDropdown componentClass="li" id="account_menu">
-          <Dropdown.Toggle useAnchor className="nav-item-iconic">
-            <Icon type="fa" name="user avatar small" />
-            {user.current_user.user.firstname} {user.current_user.user.lastname}
-          </Dropdown.Toggle>
-          <Dropdown.Menu>
-            {user.user_dropdown[0].children.map((item, i) =>
-                (item.type === 'divider' ? (
-                  <MenuItem key={i} divider />
-                ) : (
-                  <MenuItem
-                    key={i}
-                    className="user_menuitem"
-                    href={item.url}
-                    onClick={() => {
-                      changeActiveMenu({ title: 'User' });
-                    }}
-                  >
-                    {__(item.name)}
-                  </MenuItem>
-                )))}
-          </Dropdown.Menu>
-        </NavDropdown>
-      )}
+    {get(user, 'current_user.user') && (
+      <NavDropdown componentClass="li" id="account_menu">
+        <Dropdown.Toggle useAnchor className="nav-item-iconic">
+          <Icon type="fa" name="user avatar small" />
+          {user.current_user.user.firstname} {user.current_user.user.lastname}
```

Comment:
another tiny nitpick:
what about using object destructing? 
```js
const user = get(user, 'current_user.user');
const { firstname, lastname } =  user;
// ...
// ...
{firstname} {lastname}
```

Reply:
@amirfefer I tried to simplify things a bit, I can't figure out how to do object destructuring inside the JSX without creating errors. I'm open to suggestions

Reply:
```js
  const userInfo = get(user, 'current_user.user');
  const { firstname, lastname } = userInfo;
``` 
works for me with no errors, just before the JSX part.

Reply:
@amirfefer I left out adding the `const { firstname, lastname } = userInfo` there because that defeats the purpose of the check. If `userInfo` is undefined or null, there will be an error trying to deconstruct it.